packages/eslint-plugin/docs/rules/comma-spacing.md

```diff
@@ -0,0 +1,135 @@
+# Enforces consistent spacing before and after commas (`comma-spacing`)
+
+Spacing around commas improves readability of a list of items. Although most of the style guidelines for languages prescribe adding a space after a comma and not before it, it is subjective to the preferences of a project.
```

Comment:
For ease of maintenance, could you please remove the majority of this doc and make it more like our newer extension rules?

Like the one you wrote for no-extra-semi: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-extra-semi.md
Or the one I'm adding for no-dupe-class-members: https://github.com/typescript-eslint/typescript-eslint/blob/1e995d532b25b9acc2e2de0d98402b08c2ec22bc/packages/eslint-plugin/docs/rules/no-dupe-class-members.md

Reply:
@bradzacher Sure.